app/controllers/projects_controller.rb

```diff
@@ -62,6 +63,46 @@ def destroy
     }
   end
 
+  private def set_location_params
+    return unless RailsDrivers.loaded.include?(:client_location_history)
+
+    @locations = @project.enrollment_location_histories.where(located_on: location_filter.range)
+    @markers = @locations.map { |l| l.as_marker_with_name(current_user) }
```

Comment:
What should go on the marker? Right now it shows name, date, "collected by" which for external forms is the name of the project, and "Most-recent contact" if it is the most recent contact

Reply:
![Screenshot 2024-09-12 at 2 13 12 PM](https://github.com/user-attachments/assets/3d6a3efc-568f-4482-a4c6-f4645426e07a)

Reply:
That all sounds good to me.